packit_service/models.py

```diff
@@ -77,24 +77,27 @@ def get_sa_session() -> Session:
     Base = declarative_base()
 
 
-class GitProject(Base):
+class GitProjectDtb(Base):
```

Comment:
now i get it, dtb stands for database, right?

Reply:
Is it ok or I need to choose some better shortcut?

Reply:
honestly, this is the first time I see "Dtb" as a shortcut for database, that's why I was so confused. Usual term I was familiar with was "Db".

In django, the best practice is to suffix with "Model".

My opinion: I don't care, to me this is picking nits - my editor will complete everything anyway :) Hence I'm clicking resolve.

Reply:
I've moved to `Model`. I am used to it from the Django world as well...